pages/book/statements.mdx

```diff
@@ -8,14 +8,18 @@ The following statements can appear anywhere in the [function](/book/functions)
 
 The `let{:tact}` statement allows local and [block](#block)-scoped variable declaration.
 
-In Tact, declaring a variable always requires an initial value and an explicit type:
+In Tact, declaring a local variable always requires an initial value. However, the type ascription can be omitted and Tact will try to infer it from the initial value:
 
 ```tact
-let value: Int = 123;           // declaration
-let valueOptional: Int? = null; // Int or null
+let value: Int = 123; // full declaration with type and value
+let vInferred = 123;  // inferred type Int
 
-let valueMap: map<Int, Int> = emptyMap();
-let valueMapWithSerialization: map<Int as uint8, Int as uint8> = emptyMap();
+let vOptional: Int? = null;           // explicit type Int or null
```

Comment:
This is a really good example! We should add some more info on `null` -- it's a polymorphic thingy (can mean an empty map of arbitrary `map<K,V>` and the missing value of an option typoe) and hence something like `let foo = null` fails. Also, we use *local* type inference, hence something like

```
let optInt = 42;
optInt = null;
```

won't work, because `optInt`'s type would be inferred to be more specific than the user intended.